benchpress/jobs/jobs.yml

```diff
@@ -71,4 +71,11 @@
     num-threads: 1
     scale-factor: 1
     runtime: 1
-
+- benchmark: nginx_wrk_bench
+  name: nginx_wrk_bench default
+  description: defaults for nginx_wrk_bench
+  args:
+    - '-c100'  # Connections
```

Comment:
Can we keep the default inputs to match what I had in the wrk run script?

Reply:
Sure, I'll update it. 👍